apply/index.html

```diff
@@ -27,7 +27,7 @@ <h2>Requirements</h2>
 				<ul>
 					<li>You must have 500+ posts on the Scratch Discussion Forums.</li><br>
 					<li>You will not be added if any of your posts made within the month contain spam, trolling, necroposts, or generally breaking the rules.</li><br>
-					<li>If you are mainly active in Collaborations, Requests, Project Ideas, Things I'm Making and Creating, Things I'm Reading and Playing, New Scratchers, or Announcements, you may not apply unless you are active in other forums also.</li>
+					<li>If you are mainly active in Collaborations, Requests, Project Ideas, Things I'm Making and Creating, Things I'm Reading and Playing, New Scratchers, Announcements or Project Save & Level Codes you may not apply unless you are active in other forums also.</li>
```

Comment:
To keep it consistent with the previous version, it should be "Announcements, or Project Save & Level Codes," instead of "Announcements or Project Save & Level Codes".

Additionally, we ask that you please only create pull requests for open issues (see our [contributing guidelines](https://github.com/theforumhelpers/theforumhelpers.github.io/blob/master/CONTRIBUTING.md) for more information) but I think we can let this slide because I've just created an issue marked as help wanted for this.